### PR TITLE
E 946 build profile panel for projects

### PR DESCRIPTION
**Pull request by alexbeno**

### CODE REVIEW

**Comment by alexbeno** [comment]
waiting from back to have project details on reward API + skeleton from product

File: .storybook/stories/GithubRepoDetails.stories.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { ComponentStory } from "@storybook/react";
 
-import GithubRepoDetails from "src/pages/ProjectDetails/Overview/GithubRepoDetails/View";
+import GithubRepoDetails from "src/components/Project/Overview/OverviewRepos/GithubRepoDetails/View";
```
Comment (pixelfact) [inline .storybook/stories/GithubRepoDetails.stories.tsx:3]: need to fix story ?

File: .storybook/stories/OverviewPanel.stories.tsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import withAuthProvider from "../decorators/withAuthProvider";
-import OverviewPanel from "src/pages/ProjectDetails/Overview/OverviewPanel";
+import OverviewPanel from "src/pages/ProjectDetails/Overview/components/OverviewPanel";
```
Comment (pixelfact) [inline .storybook/stories/OverviewPanel.stories.tsx:2]: need to fix story ?

File: src/App/Stacks/ProjectOverviewSidePanel/ProjectOverviewSidePanel.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import { ProjectOverviewInformations } from "./components/Informations";
+import { ProjectOverviewReposCard } from "./components/Repos";
+import MarkdownPreview from "src/components/MarkdownPreview";
+
+type Props = {
+  slug: string;
+};
+
+export const ProjectOverviewSidePanel = ({ slug }: Props) => {
+  const { data: project, isLoading } = ProjectApi.queries.useGetProjectBySlug({
```
Comment (pixelfact) [inline src/App/Stacks/ProjectOverviewSidePanel/ProjectOverviewSidePanel.tsx:12]: this is a reminder for me to give you the appropriate skeleton

File: src/libs/react-stack/components/History.tsx [hunk 4]
```diff
@@ -29,6 +29,8 @@ export const History = () => {
     };
   }, [history]);
 
+  console.log("history", history);
```
Comment (pixelfact) [inline src/libs/react-stack/components/History.tsx:32]: Oupsii

File: src/App/Stacks/GithubWorkflow/ClaimSidePanel/ClaimSidePanel.tsx [hunk 5]
```diff
@@ -94,7 +94,7 @@ export default function ClaimSidePanel({ projectSlug }: ClaimSidePanelProps) {
 
   return (
     <div className="flex h-full flex-col justify-between">
-      <div className="flex h-full flex-col px-4 py-8">
+      <div className="flex h-full flex-col px-4 py-8 pt-0">
```
Comment (haydencleary) [inline src/App/Stacks/GithubWorkflow/ClaimSidePanel/ClaimSidePanel.tsx:97]:
```suggestion
      <div className="flex h-full flex-col px-4 pb-8">
```

File: src/App/Stacks/GithubWorkflow/TutorialSidePanel/TutorialSidePanel.tsx [hunk 6]
```diff
@@ -16,7 +16,7 @@ export default function TutorialSidePanel() {
 
   return (
     <div className="flex h-full flex-col justify-between">
-      <div className="flex h-full flex-col px-4 py-8">
+      <div className="flex h-full flex-col px-4 py-8 pt-0">
```
Comment (haydencleary) [inline src/App/Stacks/GithubWorkflow/TutorialSidePanel/TutorialSidePanel.tsx:19]:
```suggestion
      <div className="flex h-full flex-col px-4 pb-8">
```

File: src/App/Stacks/ProjectOverviewSidePanel/ProjectOverviewSidePanel.tsx [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+  }
+
+  return (
+    <div className="h-full overflow-hidden py-8 pr-2 pt-12 lg:pt-0">
+      <div className="w-full px-6 pr-4">
+        <ProjectOverviewHeadersCard project={project} />
+      </div>
+      <div className="flex h-full flex-auto flex-col gap-3 overflow-auto p-px pb-6 scrollbar-thin scrollbar-thumb-white/12 scrollbar-thumb-rounded scrollbar-w-1.5">
+        <div className="flex flex-col gap-4 px-6 pr-4">
+          {project.longDescription && (
```
Comment (haydencleary) [inline src/App/Stacks/ProjectOverviewSidePanel/ProjectOverviewSidePanel.tsx:32]: Doesn't this render the long description + jsx ?.

File: src/components/Project/Overview/OverviewLead.tsx [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+  const showInvited = isProjectLeader || roles.includes(HasuraUserRole.Admin);
+
+  return filteredLeads.length > 0 ? (
+    <Section
+      testId="project-leads"
+      icon={SectionIcon.Star}
+      title={T("project.details.overview.projectLeader", { count: filteredLeads.length })}
+    >
+      <div className="flex flex-row flex-wrap gap-3">
+        {(filteredLeads || []).map(lead => (
```
Comment (haydencleary) [inline src/components/Project/Overview/OverviewLead.tsx:30]: Wouldn't this be better? Same below on line 43.

```suggestion
        {filteredLeads?.map(lead => (
```

**Review by haydencleary — APPROVED** [review]

**Pull request merged**